Added doc for dry-run for file kind for Github (#2005)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/github/github.md

```
:::

#### Dry-run for file changes

> To prevent unwanted changes to the ingested file, you can use the dry-run mechanism utilizing GitHub Check. 
> When `validationCheck: true` is enabled in the file kind mapping, our Github app will run schema validation on these files before they are processed.

Reviewer: ```suggestion
To prevent unwanted changes to the ingested file, you can enable `GitHub checks` to perform a validation on ingested files.  
When `validationCheck: true` is enabled in the `kind: file` mapping, Port's Github app will perform a schema validation on these files before they are processed.
```
